revoke/revoke.go

```diff
@@ -22,14 +24,86 @@ import (
 	"github.com/cloudflare/cfssl/log"
 )
 
-// HardFail determines whether the failure to check the revocation
-// status of a certificate (i.e. due to network failure) causes
-// verification to fail (a hard failure).
-var HardFail = false
+// Revoke type contains configuration for each new revoke instance
+type Revoke struct {
+	// LocalCRL contains path to the local CRL file. When set, certificate
+	// will be checked only using local CRL script, remote methods will be
+	// skipped.
+	localCRL string
+	// HardFail determines whether the failure to check the revocation
+	// status of a certificate (i.e. due to network failure) causes
+	// verification to fail (a hard failure).
+	hardFail bool
+	// crlSet associates a PKIX certificate list with the URL the CRL is
+	// fetched from.
+	crlSet map[string]*pkix.CertificateList
+	// lck is a Mutex locker for the struct
+	lck sync.Mutex
```

Comment:
It's somewhat picky, but can this be `lock`? It's one extra character and it's more readable.